### ISSUE TITLE
About gan training

### ISSUE THREAD

**Issue by lhao0301** [XPixelGroup/HYPIR#34]
This is not a issue about the repo, but a problem I met during using gan-loss in similar restoration task. 

I use the `D-Net`, which load the pretrained `Decoder` in #23, as my discriminator. Generator is a pretrained model for restoration. I use new data to finetune them together. The logits curve for real and fake batch data are not similar as it in paper. Logits between fake data and real data are of great difference.
Any suggestions are helpful and thankful.

my training:
<img width="1736" height="981" alt="Image" src="https://github.com/user-attachments/assets/9ebb2aef-7dde-4157-aa75-3492e8892bb0" />

curve in paper:
<img width="1074" height="466" alt="Image" src="https://github.com/user-attachments/assets/a48327fa-9c46-42ca-90b6-519b3ba64dff" />

**Comment by feature-space-move** [comment]
> thanks for your suggestions.
> 
> 1. this analysis is reasonable. I will try to make the generator stronger. I find that the generator can not fit the patch pairs in training set well. Did u met similar problem during your training?
> 2. I also do training from scratch(discriminator) in that I suspect the discriminator is too strong. And I find that after several(~30) iterations, the gap between `real_logits` and `fake_logits` grows wider quickly.

Hello, I met the same problem as yours. Whether I train my custom document images or public landscape images, the D_ real_logits and D_ fake_logits almost always grows wider. I wonder how do you solve it?
